frontend/components/TableSkeleton.tsx

```diff
@@ -0,0 +1,35 @@
+import { Skeleton } from "@/components/ui/skeleton";
+import { Table, TableBody, TableCell, TableRow } from "@/components/ui/table";
+
+export default function TableSkeleton() {
+  return (
+    <div className="bg-white">
+      <Table className="caption-top not-print:max-md:grid">
```

Comment:
```suggestion
      <Table className="not-print:max-md:grid">
```
Shouldn't do anything here.